server/neptune/workflows/internal/notifier/github.go

```diff
@@ -196,3 +204,7 @@ func determineCheckRunState(workflowState *state.Workflow) github.CheckRunState
 func waitingForActionOn(job *state.Job) bool {
 	return job != nil && job.Status == state.WaitingJobStatus && len(job.OnWaitingActions.Actions) > 0
 }
+
+func BuildCheckRunTitle(mode string, rootName string) string {
```

Comment:
I think i like the idea of just having two separate functions: `BuildDeployCheckRunTitle` `BuildPlanCheckRunTitle` since you're not going to have mode everywhere and now we need to know to use "deploy" in a few places.

Reply:
done